src/gsi/setupcldtot.F90

```diff
@@ -273,7 +273,7 @@ subroutine setupcldtot(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_di
          allocate(cdiagbuf(nobs*nsig),rdiagbuf(nreal,nobs*nsig))
          rdiagbuf=zero
      endif
-     if (i_cloud_q_innovation == 2 .or. i_cloud_q_innovation == 3) then
+     if (i_cloud_q_innovation .ge. 20 .or. i_cloud_q_innovation == 3) then
```

Comment:
Need to change ".ge." to ">=".
This is required by the GSI code standard.

Reply:
updated in another commit